static/i18n/en/app.json

```diff
@@ -892,7 +892,9 @@
         "title": "Account",
```

Comment:
Only update English language file. the rest is handled by crowdin, a github plugin that creates and handles the translated files.

Reply:
ok